android-app/src/main/java/org/example/app/view/ConfigActivity.kt

```diff
@@ -24,10 +27,12 @@ class ConfigActivity :
     override val viewModelClass: Class<ConfigViewModel> = ConfigViewModel::class.java
     override val viewModelVariableId: Int = BR.viewModel
 
+    @Inject lateinit var factory: SharedFactory
```

Comment:
i think will be better if we will inject here not `SharedFactory` but `ConfigFactory`

Reply:
done